[2025-01-31T00:03:08.522+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T00:03:08.542+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-01-31T00:03:01.016768+00:00 [queued]>
[2025-01-31T00:03:08.552+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-01-31T00:03:01.016768+00:00 [queued]>
[2025-01-31T00:03:08.554+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-31T00:03:08.568+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-01-31 00:03:01.016768+00:00
[2025-01-31T00:03:08.575+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=18591) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T00:03:08.577+0000] {standard_task_runner.py:72} INFO - Started process 18593 to run task
[2025-01-31T00:03:08.577+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_file_in_directory', 'manual__2025-01-31T00:03:01.016768+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpk1orl1tz']
[2025-01-31T00:03:08.578+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask check_file_in_directory
[2025-01-31T00:03:08.795+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-01-31T00:03:01.016768+00:00 [running]> on host f85a165ecee3
[2025-01-31T00:03:08.874+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T00:03:01.016768+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T00:03:01.016768+00:00'
[2025-01-31T00:03:08.875+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T00:03:08.876+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T00:03:08.876+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-01-31 00:03:08.543907+00:00
[2025-01-31T00:03:08.877+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-01-31T00:03:08.878+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T00:03:08.878+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-31T00:03:08.883+0000] {python.py:240} INFO - Done. Returned value was: read_files
[2025-01-31T00:03:08.883+0000] {branch.py:38} INFO - Branch into read_files
[2025-01-31T00:03:08.884+0000] {skipmixin.py:230} INFO - Following branch ('read_files',)
[2025-01-31T00:03:08.894+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-01-31T00:03:08.921+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T00:03:08.922+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_file_in_directory, run_id=manual__2025-01-31T00:03:01.016768+00:00, execution_date=20250131T000301, start_date=20250131T000308, end_date=20250131T000308
[2025-01-31T00:03:08.931+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T00:03:08.932+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T00:03:08.933+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-01-31 00:03:01.028132+00:00
[2025-01-31T00:03:08.933+0000] {logging_mixin.py:190} INFO - Task hostname:f85a165ecee3 operator:BranchPythonOperator
[2025-01-31T00:03:08.954+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T00:03:09.121+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T00:03:09.125+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
